conans/client/installer.py

```diff
@@ -207,11 +207,17 @@ def install(self, deps_graph, remotes):
         install_graph.raise_errors()
         install_order = install_graph.install_order()
 
+        package_count = sum([sum(len(install_reference.packages.values())
+                                for level in install_order
+                                for install_reference in level)])
+        installed_count = 1
```

Comment:
Who is incrementing the count?

Reply:
Oops nice catch, it got transpapelado between git stashes